doc/src/build/fullnode.md

```diff
@@ -288,3 +288,119 @@ If you followed the instructions for [Building from Source](#building-from-sourc
     ```
 
 Your Full node starts on: `http://127.0.0.1:9000`.
+
+# Pruning 
+
+## Object Pruning
+
+Object pruning involves the removal of older object versions from the fullnode db. 
+New object versions are added to the db during transaction execution (making older versions ready for garbage collection) and can become a 
+problem if not pruned, as it can slow down the performance of the db and take up too much disk space. 
+Object pruning works in the background by tailing checkpoint stream, identifying each
+checkpoint which contains object versions which are now eligible for pruning.
```

Comment:
what does "tailing checkpoint stream" mean? 
```suggestion
Sui identifies the objects that are eligible for pruning in each checkpoint, and then performs the pruning in the background.
```

Reply:
i meant to day - it works by continuously reading checkpoints in the background but you captured it much better in words